File: backend/semgrep_rules/tests/java/kisa_sw_33_unlimited_auth_attempts.java
class kisa_sw_33_unlimited_auth_attempts {
    Object login(AuthManager auth, Object token) {
        // ruleid: kisa.sw33.java.login-handler-rate-limit-review
        return auth.authenticate(token);
    }
    Object health() {
        // ok: kisa.sw33.java.login-handler-rate-limit-review
        return "ok";
    }
    interface AuthManager { Object authenticate(Object token); }
}

File: backend/semgrep_rules/tests/python/kisa_sw_33_unlimited_auth_attempts.py
def login(request):
    # ruleid: kisa.sw33.python.login-handler-rate-limit-review
    user = authenticate(username=request.POST.get("username"), password=request.POST.get("password"))
    return user

def health(request):
    # ok: kisa.sw33.python.login-handler-rate-limit-review
    return "ok"
